vars/dockerExecute.groovy

```diff
@@ -265,19 +265,19 @@ private getDockerOptions(Map dockerEnvVars, Map dockerVolumeBind, def dockerOpti
     ]
     def options = []
     if (dockerEnvVars) {
-        for (String k : dockerEnvVars.keySet()) {
+        dockerEnvVars.keySet().each { String k ->
```

Comment:
I guess we should avoid also the keySet()
Could be done with
```
dockerEnvVars.each {k, v ->
  options.add ...
}
```
if I am not completely mistaken ...